docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -9,17 +9,41 @@
+	| `City`       | City where the site is located                   |                                                                                              |
+	| `StateID`    | Identifier of the state where the site is located|                                                                                              |
+	| `ZIP`        | ZIP code of the site                             |                                                                                              |
 	| `Phone1`     | Phone number 1                                   |                                                                                              |
 	| `Phone2`     | Phone number 2                                   |                                                                                              |
 	| `Contact1`   | Contact person 1                                 |                                                                                              |
 	| `Contact2`   | Contact person 2                                 |                                                                                              |
-	| `Alias`      | Shortname of the site. Limited to 3 characters | This field affects the alias section of the PSCID of candidates                              |
+	| `Alias`      | Shortname of the site. Limited to 3 characters   | This field affects the alias section of the PSCID of candidates                              |
 	| `MRI_alias`  | MRI specific shortname. Limited to 4 characters  |                                                                                              |
-	| `Account`    |                                                  | Deprecated                                                                                              |
-	| `Study_site` | Does this site recruit candidates?             | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+	| `Account`    |                                                  | Deprecated                                                                                   |
+	| `Study_site` | Does this site recruit candidates?               | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+
+- Table: `visit`
+
+ 	*This table stores the list of timepoints configured for the study.*
+ 	
+ 	|     Field    |          Description          |                                    Notes                                    |
+	|:------------:|:------------------------------------------------:|:--------------------------------------------------------:|
+	| `VisitID`   | Identifier of the Timepoint    | Avoid setting this field explicitly when inserting data, it auto increments.|
+	| `VisitName` | Back end name of the Timepoint | This name is used in file and imaging uploads                               |
```

Comment:
is it the same as the Visit label?

Reply:
yes new nomenclature I could add that in the notes